core/lib/l1_contract_interface/src/i_executor/structures/commit_batch_info.rs

```diff
@@ -7,6 +9,16 @@ use zksync_types::{
 
 use crate::Tokenizable;
 
+/// Loads KZG settings from the file system.
+pub fn load_kzg_settings() -> KzgSettings {
+    static KZG_SETTINGS: Lazy<KzgSettings> = Lazy::new(|| {
+        let zksync_home = std::env::var("ZKSYNC_HOME").unwrap_or_else(|_| ".".into());
```

Comment:
I don't think it's a good idea to make `l1_contract_interface` depend on `ZKSYNC_HOME` (as it's an otherwise pretty much a pure crate that can be used anyhwere).

You can move this logic to the place that actually uses it (I guess commitment generator).